File: Heavyweights/brigitte/Brigitte.java

public class Brigitte {
    private int stamina = 100;
    private boolean auraActive = true;

    public void paralyzingSmite() {
        System.out.println("Brigitte strikes! Enemy slowed for 2 seconds.");
    }

    public void shieldSmite() {
        System.out.println("Shield raised! Blocking attacks and creating protective barrier.");
    }

    public void northernVortex() {
        System.out.println("Enemies blinded and disoriented!");
    }

    public void uppsalaHalls() {
        System.out.println("Brigitte charges forward, knocking down and immobilizing enemies!");
    }

    public void passiveAura() {
        if (auraActive) {
            System.out.println("Protective Aura is active: bonus armor and magic resist granted.");
        }
    }
}

File: Heavyweights/brigitte/brigitte.py

class Brigitte:
    def __init__(self):
        self.stamina = 100
        self.aura_active = True

    def paralyzing_smite(self):
        print("Brigitte strikes! Enemy slowed for 2 seconds.")

    def shield_smite(self):
        print("Shield raised! Blocking attacks and creating protective barrier.")

    def northern_vortex(self):
        print("Enemies blinded and disoriented!")

    def uppsala_halls(self):
        print("Brigitte charges forward, knocking down and immobilizing enemies!")

    def passive_aura(self):
        if self.aura_active:
            print("Protective Aura is active: bonus armor and magic resist granted.")
